README.md

```diff
@@ -165,6 +165,10 @@ Be sure to have all those installed:
 
 Create a `.env` file containing the following variables:
 
+- `API_BASE_PATH`
+
+  This is the base path that will be used to build the API URL to call homer, for example `https://homer.com/<API_BASE_PATH>/command`.
```

Comment:
```suggestion
  This is the base path that will be used to build the API URL to call homer, for example `https://homer.com<API_BASE_PATH>/command`.
```

That variable is supposed to start with a slash, right?

Reply:
Good catch, thanks.